libsolc/libsolc.cpp

```diff
@@ -299,4 +299,17 @@ extern char const* compileStandard(char const* _input, CStyleReadFileCallback _r
 	s_outputBuffer = compileStandardInternal(_input, _readCallback);
 	return s_outputBuffer.c_str();
 }
+extern char const* solidity_license()
+{
+	return license();
+}
+extern char const* solidity_version()
+{
+	return version();
+}
+extern char const* solidity_compile(char const* _input, CStyleReadFileCallback _readCallback)
```

Comment:
@chfast could we just use `alias` here ?

Reply:
There is no such thing as alias for function in C++. You can have a `constexpr` pointer to other function locally, but that is something different (will not be exported as a function).

1. `extern` keyword is wrong here.
2. You should move the implementation to `solidity_license()` and make the local alias `license()`. See example in ethash library: https://github.com/chfast/ethash/blob/master/include/ethash/ethash.hpp#L52-L53.

Reply:
Yeah that will be a next step, but for now this is fine.